users/tests.py

```
@@ -0,0 +1,37 @@
+from django.test import TestCase
+from users.models import CustomUser
+
+class CustomUserTestCase(TestCase):
+    def setUp(self):
+        self.u1 = CustomUser.objects.create_user('test_user_1', 'user_1@test.com', 'lubieplacki')
+        self.u2 = CustomUser.objects.create_user('test_user_2', 'user_2@test.com', 'lubieplacki')
+        self.u3 = CustomUser.objects.create_user('test_user_3', 'user_3@test.com', 'lubieplacki')
+        
+    def test_follow_user(self):
+        self.u1.follow_user(self.u2)
+        self.assertEqual(self.u2.get_my_followers()[0], self.u1)
+
+    def test_unfollow_user(self):
+        self.u1.follow_user(self.u2)
+        if self.u1 in self.u2.get_my_followers():
```

you shouldn't use if and return in tests, you should do just plain assertions instead of adding logic.
What I mean is that the first line is adding u1 to u2 followers and then you check if u1 is in u2 followers, if for some reasons you won't find u1 in u2 followers this test should fail loudly as it means that something is broken along the road

---

which is crashing in that case, but asserting that False is True